tuf/data/targets.go

```diff
@@ -59,22 +57,11 @@ func (t SignedTargets) GetMeta(path string) *FileMeta {
 // order to determine priority.
 func (t SignedTargets) GetDelegations(path string) []*Role {
```

Comment:
Non-blocking - I think nothing actually calls this - can this just be removed or will we need it for a future use?